phys2bids/bids.py

```diff
@@ -290,3 +291,99 @@ def readme_file(outdir):
         LGR.warning('phys2bids could not find README,'
                     'generating it EMPTY, please fill in the necessary info')
         utils.write_file(file_path, '', text)
+
+
+def update_bids_name(basename, **kwargs):
```

Comment:
eh, with https://github.com/nipy/heudiconv/pull/544 I feel that we are doomed to start some `minibids` or `bidsnonofficial` or `bidspy` (so it is not `pybids`), whatever,  to start with a basic schema-driven bids filenames support and may be later expanded to something else (validator etc).  otherwise -- we keep breeding duplicated code across the ecosystem, not good!  WDYT @tsalo ?  I can move that heudiconv PR into its own python package, will you join the fun? ;)